test/common/schemaform/review/ReviewCollapsibleChapter.unit.spec.jsx

```diff
@@ -126,9 +132,9 @@ describe('<ReviewCollapsibleChapter>', () => {
         viewedPages={new Set()}
         onEdit={onEdit}
         open
-        pages={pages}
+        expandedPages={pages}
         chapterKey={chapterKey}
-        chapter={chapter}
+        chapterFormConfig={chapter}
```

Comment:
Since I used something called `chapter` to represent the array of objects that `ReviewChapters` iterates over to create an array of `ReviewCollapsibleChapter`s, decided to rename this so that there was no confusion about what is getting passed in.